recipes/proj/all/patches/use-cmake-targets-7.2.0.patch

```diff
@@ -0,0 +1,25 @@
+--- a/src/lib_proj.cmake
++++ b/src/lib_proj.cmake
+@@ -405,19 +405,16 @@ if(USE_THREAD AND Threads_FOUND AND CMAKE_USE_PTHREADS_INIT)
+   target_link_libraries(${PROJ_CORE_TARGET} ${CMAKE_THREAD_LIBS_INIT})
+ endif()
+ 
+-target_include_directories(${PROJ_CORE_TARGET} PRIVATE ${SQLITE3_INCLUDE_DIR})
+-target_link_libraries(${PROJ_CORE_TARGET} ${SQLITE3_LIBRARY})
++target_link_libraries(${PROJ_CORE_TARGET} CONAN_PKG::sqlite3)
+ 
+ if(TIFF_ENABLED)
+   target_compile_definitions(${PROJ_CORE_TARGET} PRIVATE -DTIFF_ENABLED)
+-  target_include_directories(${PROJ_CORE_TARGET} PRIVATE ${TIFF_INCLUDE_DIR})
+-  target_link_libraries(${PROJ_CORE_TARGET} ${TIFF_LIBRARY})
++  target_link_libraries(${PROJ_CORE_TARGET} TIFF::TIFF)
+ endif()
+ 
+ if(CURL_ENABLED)
+   target_compile_definitions(${PROJ_CORE_TARGET} PRIVATE -DCURL_ENABLED)
+-  target_include_directories(${PROJ_CORE_TARGET} PRIVATE ${CURL_INCLUDE_DIR})
+-  target_link_libraries(${PROJ_CORE_TARGET} ${CURL_LIBRARY})
++  target_link_libraries(${PROJ_CORE_TARGET} CURL::libcurl)
```

Comment:
Could this be applied to the older patches? https://github.com/conan-io/conan-center-index/blob/43fd47439ea0b73ab742d6b1d21de0522a638958/recipes/proj/all/patches/use-cmake-targets-7.1.1.patch#L22

Reply:
done